site/app/controllers/DockerInterfaceController.php

```diff
@@ -57,19 +63,12 @@ public function showDockerInterface(): MultiResponse {
             )
         );
     }
-    /**
-     * @return JsonResponse | MultiResponse
-     */
+
     #[Route("/admin/add_image", methods: ["POST"])]
     #[Route("/api/admin/add_image", methods: ["GET"])]
-    public function addImage() {
+    public function addImage(): JsonResponse {
         $user = $this->core->getUser();
-        if (is_null($user) || !$user->accessFaculty()) {
```

Comment:
Is there another place where user access is checked, or do we want this page to be accessible to anyone?

Reply:
Although we do have several access control before add image, we should still have one. I have just added jsonreponse check, PR #7905 by Chris also did same check for `updateDockerCall` function so followed same structure.